### PR TITLE
Support importing unknown assets

### PR DESCRIPTION
**Pull request by deanshub**
### 🔦 Summary
in reference to #821 and #942
this feature will allow users to require any file,
the files with the unknown extensions will be caught by the [url loader](https://github.com/wix/yoshi/compare/support-unknown-extensions?expand=1#diff-b91ad4512169022152dc9db9e5f4abcaR562)

### 🗺️ Test Plan
importing unknown file extensions and testing if they got into the app using kitchensink tests

### CODE REVIEW

File: packages/yoshi/config/webpack.config.js [hunk 1]
```diff
@@ -546,17 +546,29 @@ function createCommonWebpackConfig({
...
+                reStyle,
+              ],
               options: {
                 name: staticAssetName,
                 limit: 10000,
               },
             },
           ],
         },
+        // ** STOP ** Are you adding a new loader?
```
Comment (ranyitz) [inline packages/yoshi/config/webpack.config.js]: This is great!

File: test/projects/kitchensink-javascript/src/components/features/unknown-inclusion.spec.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import UnknownInclusion from './unknown-inclusion';
+
+it('should pass', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<UnknownInclusion />, div);
```
Comment (ranyitz) [inline test/projects/kitchensink-javascript/src/components/features/unknown-inclusion.spec.js]: Maybe I'm missing something, but in this case, this is not an `unknown` but rather a specific extension added to our jest preset's list of known extensions (`webp`)

Comment (deanshub) [reply test/projects/kitchensink-javascript/src/components/features/unknown-inclusion.spec.js]: I needed a file extension that we have inside our preset so I won't have to eject from jest-yoshi-preset so tests will pass, but still isn't in our webpack config extensions for url-loader
any thoughts around that?

**Comment by deanshub** [comment]
> The support for unknown extensions is currently added only to the webpack loader, but not to the test environment.
My thoughts around that is that we should still get issues\PRs for common extensions but for the rest of the extensions, the specific project can and should add a mapper\transformer of their own

File: test/projects/kitchensink-javascript/integration/loaders.test.js [hunk 3]
```diff
@@ -324,4 +324,16 @@ describe('webpack', () => {
       expect(result).toBe('This is an abstract.');
     });
   });
+
+  describe('unknown', () => {
```
Comment (ranyitz) [inline test/projects/kitchensink-javascript/integration/loaders.test.js]: This is the same test as the image inclusion test, here we specifically testing `webp` but it is the same as any other extension. I would prefer removing this test.

**Pull request closed without merging**